File: Exercicios-Beecrowd/Python/1040.py
a,b,c,d= input().split(" ")
a= float(a)
b= float(b)
c= float(c)
d= float(d)
media= ((a*2) + (b*3) + (c*4) + (d))/10
print(f'Media: {media:.1f}')
if media>= 7:
    print('Aluno aprovado.')
elif media<5:
    print('Aluno reprovado.')
elif media >=5 and media<6.9:
     print('Aluno em exame.')
     exame= float(input())
     print(f'Nota do exame: {exame:.1f}')
     media_final = (exame+media)/2
     if media_final>=5 :print('Aluno aprovado.')
     if media_final<=4.9: print('Aluno reprovado.')
     print(f'Media final: {media_final:.1f}')

File: Exercicios-Beecrowd/Kotlin/1040.kt
import java.util.*
import kotlin.math.floor

fun main(args: Array<String>) {

    val entrada= readLine().toString().split(" ")
    val n1=entrada[0].toDouble()
    val n2=entrada[1].toDouble()
    val n3=entrada[2].toDouble()
    val n4=entrada[3].toDouble()
    val mediaa= ((n1*2)+(n2*3)+(n3*4)+n4)/10
    val media=floor(mediaa*10)/10
    println("Media: ${"%.1f".format(media).replace(",",".")}")
    if ((media)>=7){
        println("Aluno aprovado.")
    }
    else if(media<5){
        println("Aluno reprovado.")

    }
    else{
        println("Aluno em exame.")
        val n5= readLine().toString().toDouble()
        println("Nota do exame: ${n5.toString().replace(",",".")}")
        if (((n5+media)/2)>=5){
            println("Aluno aprovado.")
        }
        else{
            println("Aluno reprovado.")
        }
        println("Media final: ${"%.1f".format((n5+media)/2).replace(",",".")}")
    }
	
}